### PR TITLE
Wishlist feature

### PR DESCRIPTION
**Pull request by anupkgurung**
wishlist

### CODE REVIEW

**Review by aniketsingh13 — APPROVED** [review]
look good to me

File: page/wishlist.html [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+    <link rel="stylesheet" href="../style/header.css">
+    <link rel="stylesheet" href="../style/commons.css">
+    <link rel="stylesheet" href="../style/wishlist.css">
+    <title>Wishlist</title>
+</head>
+<body>
+    <div>
+        <div id="header" class="header-bar">
+            <div class="topbar">
+                <div><img class="logo" src="../image/logo.jpg" alt=""></div>
```
Comment (Aditi0505) [inline page/wishlist.html:20]: Please add alt description

**Review by Aditi0505 — APPROVED** [review]
LGTM

File: page/wishlist.html (hunk 1 above)
Comment (anupkgurung) [reply page/wishlist.html:20]: sure !!

**Pull request merged**